codility_training/lessons.lesson10.PrimeAndCompositeNumbers.CountFactors/sangmandu.py

```diff
@@ -0,0 +1,25 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(N):
+    # write your code in Python 3.6
+    pass
+
+    i = 1
+    limit = 0
+    while i * i <= N:
+        limit += 1
+        i += 1
+
+    i -= 1
+    cnt = 0
+    for j in range(1, limit + 1):
+        if N % j == 0:
+            cnt += 1
+        j += 1

```

Comment:
```suggestion
    i = 1
    while i * i <= N: i += 1
    i -= 1
    
    cnt = 0
    for j in range(1, i + 1):
        cnt += 1 if N % j == 0 else 0
        j += 1
```
이렇게 표현할 수 있습니다.
limit은 굳이 필요 없을 것 같아요~